### ISSUE TITLE
Offset nametags

### ISSUE THREAD

**Issue by ChaiTeaNunes** [ironbane/ironbane-client#18]
![screen shot 2015-01-09 at 9 23 40 pm](https://cloud.githubusercontent.com/assets/5761439/5690559/ca8ba396-9846-11e4-8038-51b7b59a567d.png)

The name tags have to be directly above the player, otherwise it looks weird to the other players.

**Comment by bsparks** [comment]
yeah that is a weirdness with rotation because of it being a child, prolly need to figure out the math or just make it not a child and have it follow the position (like the camera)
